src/Magnum/ImGuiIntegration/Integration.h

```diff
@@ -42,31 +42,81 @@ namespace Magnum { namespace ImGuiIntegration {
 
 class MAGNUM_IMGUIINTEGRATION_EXPORT ImGuiShader : public GL::AbstractShaderProgram {
```

Comment:
Does this even need to be exposed publicly? For Magnum::Ui I'm not really exposing the shaders either. 

 Then it wouldn't need any docs (and also any `EXPORT`, I think).

Reply:
I was thinking of moving it into an `Implementation` namespace even... would that make sense? Would more clearly convey that it's not API

Reply:
Yes, definitely.

Reply:
Also: unless you construct / destruct this class (or call any non-inline functions on it) in the public header (which looks like you don't), there's no need to `MAGNUM_IMGUIINTEGRATION_EXPORT` it.